yarn.lock

```diff
@@ -23803,6 +23803,11 @@ tinycolor2@^1.4.1:
   resolved "https://registry.yarnpkg.com/tinycolor2/-/tinycolor2-1.4.1.tgz#f4fad333447bc0b07d4dc8e9209d8f39a8ac77e8"
   integrity sha1-9PrTM0R7wLB9TcjpIJ2POaisd+g=
 
+tinycolor2@^1.4.2:
```

Comment:
Why was our old version not in the yarn.lock?

Reply:
Also... Where is the jquery dependency? I was just looking at https://bundlephobia.com/package/tinycolor2@1.4.1

Reply:
Looks like the jquery dependency was just for its demo. So I don't think it was actually packaged into our bundles

Reply:
We still have a dependency on 1.4.1 because the storybook uses a component called react-color which depends on the version of tinycolor2 in version 1.4.1 🙂, we can evaluate updating the storybook later to harvest new features and maybe not depend on 1.4.1 anymore. As Storybook dependencies don't impact in the end, so we don't have any problems.

Reply:
Apparently, it actually doesn't seem to have a jquery dependency but I can only see jquery removal in demos https://github.com/bgrins/TinyColor/commit/250a1e2421242b336770d50c2b5e1eae292bc727 🤷‍♂️

Reply:
Your last message only appeared to me later 🤦‍♂️